enterprise/api/v1/serializers.py

```diff
@@ -721,12 +721,25 @@ class Meta:
         )
 
     user = UserSerializer()
-    enterprise_customer = EnterpriseCustomerSerializer()
     data_sharing_consent_records = serializers.SerializerMethodField()
     groups = serializers.SerializerMethodField()
     role_assignments = serializers.SerializerMethodField()
     enterprise_group = serializers.SerializerMethodField()
 
+    def to_representation(self, instance):
```

Comment:
could you do this via a `SerializerMethodField()` instead?  It'd be a little more explicit, and it would help with auto-discovery of fields in tools like drf-yasg or drf-spectacular.

Reply:
I did try this originally, but seemed to run into an issue where it would actually end up breaking the `enterprise-customer` list/detail APIs (insofar as the tests go), regarding the `enterprise_notification_banner` suddenly starting to return `None` instead of the current empty state dict.

I will try again, though, as I'm blanking if I was properly passing the `enterprise_customer` explicitly through an `instance` kwarg or not.

The `to_representation` was a pivot away from `SerializerMethodField`. I agree it would be preferred; will update soon.

Reply:
Updated to use `SerializerMethodField`. When I originally tried it, I wasn't passing `context=self.context`, but ensuring its passed works fine!